fbw-a380x/src/systems/instruments/src/SD/Pages/Cruise/elements/CruisePressure.tsx

```diff
@@ -20,18 +19,10 @@ export const CruisePressure = () => {
     const deltaPress = splitDecimals(deltaPsi);
 
     useEffect(() => {
-        if (landingElevDialPosition === 0) {
-            // On Auto
-            const nearestfifty = Math.round(landingRunwayElevation / 50) * 50;
-            setLdgElevValue(landingRunwayElevation > -5000 ? nearestfifty.toString() : 'XX');
-            setCssLdgElevName(landingRunwayElevation > -5000 ? 'Green' : 'Amber');
-        } else {
-            // On manual
-            const nearestfifty = Math.round(landingElev / 50) * 50;
-            setLdgElevValue(nearestfifty.toString());
-            setCssLdgElevName('Green');
-        }
-    }, [landingElevDialPosition, landingRunwayElevation]);
+        const ldgElevValue = Math.round(landingElev.value / 50) * 50;
+        setLdgElevValue(ldgElevValue.toString());
```

Comment:
This is creating a loop, right, since the useEffect is depending on the same state?

Reply:
Yes, thanks! Changed now.